server/db/index.js

```diff
@@ -3,7 +3,18 @@
 const db = require('./database');
 const { Users, Orders, OrderDetails, Products, Wishlist } = require('./models/index');
 
-//set up foreign keys here, i.e. connect models
+//set up associations here
+//Wishlist gets the association key as userId
+Users.hasOne(Wishlist);
```

Comment:
If wishlist is a list of products by userid, then can't a user be listed in the wishlist table multiple times?

Reply:
I changed it to hasMany as well.